docs/markdown/snowplow_web_overview.md

```diff
@@ -596,7 +596,15 @@ All the incremental models in the Snowplow web package have recommended cluster
 
 ### Overriding Macros
 
-Both the cluster key macros (see above) and the `allow_refresh()` macro can be overridden. These are both [dispatched macros](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch) and can be overridden by creating your own version of the macro and setting a project level dispatch config. More details can be found in [dbt's docs](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch#overriding-package-macros)
+The cluster key macros (see above), the `allow_refresh()` and the `filter_bots` macro can be overridden. These are all [dispatched macros](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch) and can be overridden by creating your own version of the macro and setting a project level dispatch config. More details can be found in [dbt's docs](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch#overriding-package-macros)
+
+
+``` yaml
+# Your_dbt_project/macros/filter_bots.sql
+{% macro default__filter_bots() %}
+  and ev.useragent not similar to '%(YOUR_CUSTOM_PATTERN|bot|crawl|slurp|spider|archiv|spinn|sniff|seo|audit|survey|pingdom|worm|capture|(browser|screen)shots|analyz|index|thumb|check|facebook|PingdomBot|PhantomJS|YandexBot|Twitterbot|a_archiver|facebookexternalhit|Bingbot|BingPreview|Googlebot|Baiduspider|360(Spider|User-agent)|semalt)%'
+{% endmacro %}
+```
```

Comment:
🚫
This file is going to be stripped down heavily from #120 , could you raise a PR in https://github.com/snowplow/documentation instead to edit the `docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-advanced-operation/index.md` file and add this there instead?

Reply:
Sure, we would probably need some marketing around this to highlight to users that the current docs are not the most comprehensive anymore, but then I guess it is part of the upcoming release around that.

Reply:
Yep 100%, the new docs also point to the docs.snowplow.io site as the main source going forward as well once they are merged.